src/scheduler/node_info.c

```diff
@@ -1174,135 +1147,6 @@ add_node_state(node_info *ninfo, char *state)
 	return 0;
 }
 
-/**
- * @brief
- *      talk_with_mom - talk to mom and get resources
- *
- * @param[in,out]	ninfo	-	the node to to talk to its mom
- *
- * @return	int
- * @return	1	: on error
- * @return	0	: on success
- *
- */
-int
-talk_with_mom(node_info *ninfo)
```

Comment:
I see all the rm stuff being removed (which includes load), so the load won't be set, but I don't see all the load code going away.  You need to remove load_balancing from conf.  You need to remove lowest_load from smp_cluster_dist.  You need to remove all the code that uses them.  You need to remove the load member from node_info and all the code which uses that.